osbs/constants.py

```diff
@@ -71,3 +71,11 @@
 # Logging format used in Atomic Reactor
 ATOMIC_REACTOR_LOGGING_FMT = \
     '%(asctime)s platform:%(arch)s - %(name)s - %(levelname)s - %(message)s'
+
+# number of retries for http requests
+HTTP_MAX_RETRIES = 3
+
+# how many seconds should request wait for in case non-critical error has occurred
+HTTP_BACKOFF_FACTOR = 2
+
+HTTP_RETRIES_STATUS_FORCELIST = [500, 502, 503, 504]
```

Comment:
Although for conflict you may need to fetch the more recent version of the object being updated. Or maybe not.